Ba console access

bernardanderson opened the pull request.

Commit:
Merged updated UserInterface from Odd

Commit:
Made the user operation input case insensitive

Commit:
Refactored UserInterface for easier testing

Commit:
Moved the printing output

bernardanderson committed:
Complete UserInterface tests and methods
The UserInterface tests and simplified the methods. The main console
output are now in Program.cs.  Placeholders have been used in the
operation calls so the branch will compile and test can be run.

bernardanderson committed:
Fixed comment

Commit:
Moved all console input/output to Program.cs

Commit:
Added a test to check stringality of returned List
The returnedOperationResult List should be converted to a string so that
it can be displayed on the Console.  An additional test was added to see
if the Name of the type was a "String"

katerebekah commented on Mathmagician.Tests/UserInterfaceTests.cs:
```diff
@@ -0,0 +1,126 @@
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class UserInterfaceTests
```
I see the UserInterface tests here. Ignore what I said in the other PR!!

katerebekah commented on Mathmagician.Tests/UserInterfaceTests.cs:
```diff
@@ -0,0 +1,126 @@
+            Assert.AreEqual("String", myUserInterface.ReturnedOperationValues().GetType().Name);
+            Assert.AreEqual("2 66 78 ", myUserInterface.ReturnedOperationValues());
+        }
+    }
```
You can add a couple unit tests that use CollectionAssert.AreEqual to determine if all the items in the lists you are comparing are equal.

Commit:
Added two additional tests to check Lists
Two addtional tests were added to check the setting/getting of Lists in
the ConsoleAccess class.  Both a AreEqual and AreNotEqual were added to
see that you won't always get a "true".

bernardanderson commented:
@katerebekah 
I added two additional CollectionAssert tests using AreEqual.  I also threw in some AreNotEqual's to make sure that they weren't giving false positives.

katerebekah commented:
:+1:

Commit:
Merge branch 'master' into ba-ConsoleAccess

Commit:
Added the correct operation methods
Since everyone was working on a separate method, the names of the methods
were not consistent.  The method calls were added once everyone's portions
were complete. We really should have been consistent with the operation
classes and should have used an interface.

VikingPaul commented:
only thing that i see is you have primes and integers, but the rest are singular. but that's not a big deal. in the program.cs you should have something like a ReadLine at the end so we can see the output. otherwise it's good.

Commit:
Added readline at end of Program.cs to see output

VikingPaul commented:
:+1:
